collectorapp.sql

```diff
@@ -30,21 +30,27 @@ CREATE TABLE `recipes` (
   `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
   `recipe` varchar(100) DEFAULT NULL,
```

Comment:
if you don't want duplicates make this unique

Reply:
Think I'm happy to have duplicates on the name as may have more than one recipe (as in link) for a given recipe name :)